suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -73,6 +73,8 @@ const discoverAccountsByDescriptorThunk = createThunk(
             const { success, payload: accountInfo } = await TrezorConnect.getAccountInfo({
                 coin: bundleItem.coin,
                 descriptor: bundleItem.descriptor,
+                useEmptyPassphrase: true,
```

Comment:
Maybe the comment explaining why to use empty passphrase should be in this file as well?

Reply:
Same as above.